arch/x86/memchunk_sse.c

```diff
@@ -0,0 +1,56 @@
+/* memchunk_sse.h -- SSE inline functions to copy small data chunks.
```

Comment:
Extension does not match filename here either...

Reply:
Fixed